polyaxon/values.yaml

```diff
@@ -490,6 +490,10 @@ rabbitmq:
             topologyKey: "kubernetes.io/hostname"
 
 docker-registry:
+  install: true
+  registryUser:
+  registryPassword:
+  externalRegistryHost:
```

Comment:
I would just call it `registryHost:`.

Reply:
I went with `externalRegistryHost` to follow the same convention as the external postgres

Reply:
Sorry my bad, I did not pay attention 👍